addons/sale_management/tests/test_sale_order.py

```diff
@@ -283,3 +285,30 @@ def test_03_template_with_discount_excluded_pricelist(self):
             "If a pricelist is set without discount included, the discount "
             "shall be computed according to the price unit and the subtotal."
             "price")
+
+    @freeze_time('2021-10-01 00:00:00')
+    def test_04_order_with_customer_invoice(self):
```

Comment:
Nice to add a test, but it should be in `account`, probably around https://github.com/odoo/odoo/blob/56f7afb253da6560cddae121f5b6cdab3e3b1c6e/addons/account/tests/test_sequence_mixin.py#L88